<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogWindow</class>
 <widget class="QMainWindow" name="LogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查看日志</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayoutMain">
    <property name="spacing">
     <number>7</number>
    </property>
    <item>
     <widget class="QLineEdit" name="searchBar">
      <property name="styleSheet">
       <string notr="true">
        QLineEdit {
          font: 10pt "PingFang SC";
          border-radius: 15px;
          border: 1px solid #E0E0E0;
          padding: 8px 15px;
          background-color: #F7F7F7;
        }
       </string>
      </property>
      <property name="placeholderText">
       <string>搜索日志...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="logArea">
      <property name="styleSheet">
       <string notr="true">
        QTextEdit {
          font: 12pt "PingFang SC";
          border: 1px solid #E0E0E0;
          border-radius: 15px;
          background-color: #FFFFFF;
          padding: 12px;
        }
       </string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutButtons">
      <property name="spacing">
       <number>7</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>保存日志</string>
        </property>
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
            font: 10pt "PingFang SC";
            color: white;
            background-color: #A5A5A5;
            border-radius: 15px;
            padding: 10px 25px;
          }
          QPushButton:hover {
            background-color: #8E8E8E;
          }
         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportButton">
        <property name="text">
         <string>导出日志</string>
        </property>
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
            font: 10pt "PingFang SC";
            color: white;
            background-color: #A5A5A5;
            border-radius: 15px;
            padding: 10px 25px;
          }
          QPushButton:hover {
            background-color: #8E8E8E;
          }
         </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">
     QStatusBar {
       background-color: #F7F7F7;
       font: 10pt "PingFang SC";
     }
    </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
